web/themes/custom/usagov/sass/_benefits-block.scss

```diff
@@ -76,6 +76,12 @@ a.usa-button.benefits-link {
   font-weight: 600;
   width: 100%;
 
+  &:hover {
+    @include u-bg('white');
+    @include u-text('blue-80v');
+    @include u-border(2px 'solid' 'cyan-50v')
```

Comment:
I used `cyan-50v` as it was the closest to the USWDS color token

Reply:
That sounds legit, and I am no css expert, but when I pull in your branch I see this output from the theme builder: 
```
node      | [22:44:28] Starting 'buildSass'...
node      | Compiling with USWDS 3.0.1
node      | node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/utilities/_border.scss
node      | Error: "`2px solid cyan-50v` is not a valid `border` token. Use valid color, units, and border-style tokens separated by commas."
node      |     ╷
node      | 128 │   @include border-n(n, $value...);
node      |     │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
node      |     ╵
node      |   node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/utilities/_border.scss 128:3  u-border()
node      |   sass/_benefits-block.scss 86:5                                                                @forward
node      |   sass/_uswds-theme-custom-styles.scss 45:1                                                     @forward
node      |   sass/styles.scss 72:1  
```

Probably the parentheses are in the wrong place. I'm out of my element but I can grep! We have this elsewhere and it's fine apparently: 

./sass/_uswds-theme-custom-styles.scss:  `box-shadow: inset 0 0 0 2px color('cyan-50v');`

Reply:
The error goes away on my local just by adding the commas between the values : @include u-border(2px, 'solid', 'cyan-50v');

Reply:
@akf thanks for finding that! not sure how i missed the error but it was missing commas in-between the props so we should be good now!